judge/migrations/0133_add_binary_data.py

```diff
@@ -0,0 +1,457 @@
+# Generated by Django 3.2.16 on 2022-12-12 14:27
+
+import django.core.validators
+import django.db.models.deletion
+from django.db import migrations, models
+
+import judge.models.contest
+import judge.models.problem
+import judge.models.profile
+
+
+class Migration(migrations.Migration):
```

Comment:
@Xyene @quantum5 we need to do the cleanup migration at some point, should we write a separate migration or just hide it here?

Reply:
Done!